beamer/tests/agent/test_request.py

```diff
@@ -418,3 +419,32 @@ def test_handling_reorgs(config, direction, token, request_manager):
             sleeper.sleep(0.1)
 
     agent.stop()
+
+
+def test_no_source_balance_for_fill(direction, config, token, request_manager):
+    requester, target = alloc_accounts(2)
+    min_source_balance = to_wei(1, "ether")
+    config.chains["l2a"].min_source_balance = min_source_balance
+    ape.chain.provider._make_request("evm_setAccountBalance", [config.account.address, "0x0"])
```

Comment:
Could you please avoid using a private method of the `provider`?

Reply:
done